app/controllers/projects_controller.rb

```diff
@@ -46,6 +42,7 @@ def show
   def new
     image_ivars
     @project = Project.new
+    @project_dates_any = false
```

Comment:
I think this should default to `true`.  Currently if you create a project and don't mess with the date settings you get a project that is only good for today.  I think a more natural default is the "Any" state.

Reply:
Fixed: Commit [06035e3](https://github.com/MushroomObserver/mushroom-observer/pull/1658/commits/06035e309b771f9fa35996ef74e4044aa909adbe)